.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
+sudo: true
+script: mvn package && mvn exec:java -Dexec.mainClass="io.in_toto.lib.App"
```

Comment:
I would expect to run just `mvn test` and that's it.

Reply:
changed. 72b39c1bc8413c97dc4331d0a026c5b7c20182cd